src/cryptonote_core/tx_pool.cpp

```diff
@@ -820,8 +820,11 @@ namespace cryptonote
     return true;
   }
   //---------------------------------------------------------------------------------
-  void tx_memory_pool::set_relayed(const epee::span<const crypto::hash> hashes, const relay_method method)
+  void tx_memory_pool::set_relayed(const epee::span<const crypto::hash> hashes, const relay_method method, std::vector<bool> &just_broadcasted)
   {
+    if (!just_broadcasted.empty())
```

Comment:
This `if` is unnecessary.

Reply:
What @vtnerd probably meant to say that `just_broadcasted.clear();` can just be called without the `if` with the same end result.

Reply:
Correct, the STL code has to do this check basically anyway internally.